### DISCUSSION TITLE
Can I get specific pairing Object in SignClient using session topic?

### DISCUSSION THREAD

**Discussion by kyuuuK** [WalletConnect/walletconnect-monorepo#3091] [Developer Support]
Hello everyone.
I'm facing the problem while develop DApp.
I have to mange multi session in one browser, because we are planning the multi connect 
so I want to get specific pairing Object using specific session data.
please let met know.
thank you :)

**Comment by finessevanes** [comment]
You could try something like:

```js
core.getPairings().find(pairing => pairing.topic === topic);
```

**Reply by kyuuuK** [reply]
@finessevanes  thank you for reply
but, I can't find that Function.
I'm using the '@walletconnect/sign-client' npm library
and there are no 'getPairings' in Client core.
if I have referred incorrect library,
please let me know. 

my code snippet is below

```
const client = SignClient.init({
  projectId,
  relayUrl,
  metadata : Object
});
const pairings = client.core.getPairings() // here
```

**Reply by finessevanes** [reply]
Sorry about that. Try:
```js
client.core.pairing.getPairings()
```

**Reply by kyuuuK** [reply]
@finessevanes hello finessevanes
The above method didn't help me. The topic of the session and the topic of the pairing didn't match, so I couldn't obtain the information using the provided code. The above method seems to rely solely on comparing strings.
